youtube_dl/extractor/odnoklassniki.py

```diff
@@ -121,6 +121,10 @@ class OdnoklassnikiIE(InfoExtractor):
         # Paid video
         'url': 'https://ok.ru/video/954886983203',
         'only_matching': True,
+    }, {
+        # Downloads only via https
+        'url': 'https://ok.ru/video/1704000096969',
```

Comment:
>Video has been blocked due to author's rights infingement

Reply:
updated the link

Reply:
https://ok.ru/video/1705664645833 downloads fine without these changes.

Reply:
for me without this change it returns:
```
[Odnoklassniki] 1705664645833: Downloading webpage
ERROR: Unable to download webpage: HTTP Error 403: Forbidden (caused by <HTTPError 403: 'Forbidden'>); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

while for some other URLs i've tried it works

Reply:
after the latest update (2020.03.01) the issue is still reproducible for me